cypress/integration/slidingTreeDom.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+Cypress.Commands.add('hideTree', (selector) => {
```

Comment:
we are not just hiding. We are hiding, then showing.

Reply:
slideTree would be better?